pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -29,7 +31,17 @@ func TestBootstrappedConstraints(t *testing.T) {
 		t.Errorf("unexpected number of constraints: found %d, wanted %d", len(bootstrappedConstraints), len(expectedConstraints))
 	}
 
+	var orderedBootstrappedConstraints []*securityapi.SecurityContextConstraints
 	for _, constraint := range bootstrappedConstraints {
+		orderedBootstrappedConstraints = append(orderedBootstrappedConstraints, &constraint)
+	}
+	sort.Sort(scc.ByPriority(orderedBootstrappedConstraints))
+
+	for i := 0; i < len(orderedBootstrappedConstraints); i++ {
+		constraint := bootstrappedConstraints[i]
```

Comment:
shouldn't you be getting the constraint from `orderedBootstrappedConstraints` to compare against `expectedConstraintNames` rather than `bootstrappedConstraints`?  You still want the counter anyway to know which expected name to check against.